src/lib/common/webnative.ts

```diff
@@ -62,12 +46,24 @@ export const isUsernameValid = async (username: string): Promise<boolean> => {
   return webnative.account.isUsernameValid(username)
 }
 
-const debouncedIsUsernameAvailable = asyncDebounce(webnative.account.isUsernameAvailable, 300)
+const debouncedIsUsernameAvailable = asyncDebounce(
+  webnative.account.isUsernameAvailable,
+  300
+)
 
-export const isUsernameAvailable = async (username: string): Promise<boolean> => {
+export const isUsernameAvailable = async (
+  username: string
+): Promise<boolean> => {
   return debouncedIsUsernameAvailable(username)
 }
 
+export const register = async (username: string): Promise<boolean> => {
+  await initialize()
```

Comment:
So for some reason I don't fully understand, if I don't call `initialize`, `register` fails.

I think it's time to move to an "initialize on load" model with a store. Next PR?

Reply:
Yep, agreed. Next PR for this, which can be all about setting up initialization and session stores. 👍